scripts/dockerfiles/Dockerfile-base

```diff
@@ -33,5 +33,11 @@ ADD configgin /opt/hcf/configgin/
 # Add rsyslog configuration
 ADD rsyslog_conf.tgz /
 
+# Fix monit's logrotate config (created in the `apt-get install monit` above) to our new log file location
+RUN sed -i 's/log/vcap\/monit/' /etc/logrotate.d/monit
+
+# Allow monit to reload during logrotate
```

Comment:
Can you please add more detail to this comment.
The info you have in the conversation is very useful.
Please explain why the policy is the way it is, why we're changing it and link back to the online resources you've found.